service/src/tests/commons.rs

```diff
@@ -15,23 +15,17 @@
 
 */
 
-use crate::{enclave_account, ensure_account_has_funds};
-use base58::ToBase58;
-use codec::Encode;
-use ita_stf::{Getter, Index, KeyPair, ShardIdentifier, TrustedCall, TrustedGetter};
-use itp_enclave_api::enclave_base::EnclaveBase;
-use log::*;
+use ita_stf::{Getter, KeyPair, TrustedGetter};
 use serde_derive::{Deserialize, Serialize};
-use sgx_crypto_helper::rsa3072::Rsa3072PubKey;
 use sgx_types::*;
-use sp_core::{crypto::AccountId32, sr25519};
 use sp_keyring::AccountKeyring;
-use std::{fs, str};
-use substrate_api_client::{rpc::WsRpcClient, Api};
+use std::str;
 
 #[cfg(test)]
 use crate::config::Config;
 #[cfg(test)]
+use ita_stf::ShardIdentifier;
+#[cfg(test)]
 use its_primitives::{
 	traits::{Block as BlockT, SignBlock},
 	types::{Block, SignedBlock},
```

Comment:
The conditional `use`s should be placed first, before the regular ones (so when the IDE adds them automatically, they a grouped correctly).

Reply:
Haha, I wasn't aware that we agreed on that, but seems useful! 👍

Reply:
I used the one that was included in the file before but now only for test. Should I change it as well as the order of the other use statements?

Reply:
My reasoning is, that we would rather depend on `itp-types`, which is at the very bottom of the dependency hierarchy, than on `ita-stf`, which contains very high-level stuff as well. So if you're touching that code anyway, I would say change them to `itp-types`.